packages/lesswrong/components/users/UsersMenu.tsx

```diff
@@ -149,7 +149,7 @@ const UsersMenu = ({color="rgba(0, 0, 0, 0.6)", classes}: {
                 <ListItemIcon>
                   <SettingsButton className={classes.icon}/>
                 </ListItemIcon>
-                Edit Settings
+                Edit Account
```

Comment:
Also renamed "Edit Settings" to "Edit Account" everywhere so that we are using a consistent name for this page